Source/WTF/wtf/DateMath.cpp

```diff
@@ -103,10 +103,16 @@ static Vector<UChar>& innerTimeZoneOverride() WTF_REQUIRES_LOCK(innerTimeZoneOve
 
 /* Constants */
 
-const std::array<ASCIILiteral, 7> weekdayName { "Mon"_s, "Tue"_s, "Wed"_s, "Thu"_s, "Fri"_s, "Sat"_s, "Sun"_s };
 const std::array<ASCIILiteral, 12> monthName { "Jan"_s, "Feb"_s, "Mar"_s, "Apr"_s, "May"_s, "Jun"_s, "Jul"_s, "Aug"_s, "Sep"_s, "Oct"_s, "Nov"_s, "Dec"_s };
 const std::array<ASCIILiteral, 12> monthFullName { "January"_s, "February"_s, "March"_s, "April"_s, "May"_s, "June"_s, "July"_s, "August"_s, "September"_s, "October"_s, "November"_s, "December"_s };
 
+
+ASCIILiteral weekdayName(std::chrono::weekday weekday)
+{
+    static const std::array<ASCIILiteral, 7> weekdayName { "Sun"_s, "Mon"_s, "Tue"_s, "Wed"_s, "Thu"_s, "Fri"_s, "Sat"_s };
```

Comment:
```suggestion
    static constexpr std::array<ASCIILiteral, 7> weekdayName { "Sun"_s, "Mon"_s, "Tue"_s, "Wed"_s, "Thu"_s, "Fri"_s, "Sat"_s };
```
Also not sure we need the `static` to generate efficient code if we have `constexpr`.

Reply:
Based on sone quick online research, I think `static` is still useful for `constexpr` local variables.